roslibrust_test/src/main.rs

```diff
@@ -0,0 +1,34 @@
+use std::borrow::Cow;
+use std::io::Write;
+use std::process::{Command, Stdio};
+
+fn main() {
+    let tokens = roslibrust_codegen::find_and_generate_ros_messages(vec!["..".into()]);
+    let source = format_rust_source(tokens.to_string().as_str()).to_string();
+    println!("{}", source);
+}
```

Comment:
Should this just be a test instead of main?

Reply:
This package should probably be renamed. My intention is that this is an internal tool, not published to crates.io which can be used to generate code for the `local_msgs` in this repo. So the lib will maintain the generated code and this bin will act as both a generator and documentation on how to write one. Does that make sense?